cli/pack/common.go

```diff
@@ -160,26 +144,26 @@ func prepareBundle(cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx) (string, error)
 	}
 
 	if len(appList) == 0 {
-		return "", fmt.Errorf("The is no apps found in instance_enabled directory")
+		return "", fmt.Errorf("There is no apps found in instance_enabled directory")
```

Comment:
```suggestion
		return "", fmt.Errorf("There are no apps found in instance_enabled directory")
```
Also, it would be nice to provide instance_enabled directory path info.

Reply:
Fixed, thank you.